modules/react-native-status/ios/RCTStatus/LogManager.m

```diff
@@ -41,18 +41,18 @@ @implementation LogManager
 #endif
 
 #if DEBUG
-    NSString *goerliNetworkDirPath = @"ethereum/goerli_rpc_dev";
+    NSString *sepoliaNetworkDirPath = @"ethereum/sepolia_rpc_dev";
```

Comment:
cc @siddarthkay @churik - is this alright or should we just move away from there? 🤔

Reply:
it will take some trial and error to figure out if indeed the `Sepolia` Directory gets created with the same naming convention as the `Goerli` one.

If you'd like to do that experiment, its upto you or you could just remove these entries and create a follow up task for adding the `sepolia_geth.log` file as part of the logs bundle we get by shaking the device.

Also very weird this is only done for iOS.
some relevant PRs in the past are : 
-> https://github.com/status-im/status-mobile/pull/14092/files
-> https://github.com/status-im/status-mobile/pull/14126/files

Reply:
sorry for the very slow response. Thanks for this Sid! 🙏